accounts/forms.py

```diff
@@ -236,6 +236,13 @@ class UsernameReminderForm(forms.Form):
 
 
 class ProfileForm(forms.ModelForm):
+    username = forms.RegexField(
```

Comment:
Is there a way we can "import" the field from the user model form so that we make sure we apply exactly the same restrictions? (the regex, etc?)

Reply:
I guess we could define a form with only that field and then extend it in both forms, but the problem is that the fields are not equal, for example the help text is not the same.

Reply:
but we could have them both have the same help text right?
or modify the help text in the `init` of the form. Maybe we could simply assign the field into a variable in the global scope and then add it to both forms? Like:

`
username_field = forms.RegexField(
    label=_("Username"),
    ...
)

class ProfileForm(forms.ModelForm):
    username = username_field

class RegistrationForm(forms.ModelForm):
    username = username_field
`

wouldn't that work? then given that we already overwrite `init` for the `ProfileForm`, we can set the help text to '' there (or keep it as is if useful to users).

Reply:
I think that could work, I will try it